lib/surface_site_web/live/events.ex

```diff
@@ -92,29 +95,174 @@ defmodule SurfaceSiteWeb.Events do
+              </#Markdown>
+
+                See the properties <LiveRedirect label="Event" to="/properties#event-property" /> section for more details about event properties.
+
+              <#Markdown>
+
+                ### Stateless component
+                The simplest case you need to pass an event is when you create a stateless component that includes an element
+                that defines a server binding (event). Since the component is stateless, it cannot handle the event by itself
+                so it needs to receive the event handler as a property.
+
+                For example, imagine a `Button` stateless component that raises an event when the user clicks on it.
+                In the following example, we create that stateless component.
+              </#Markdown>
+
+              <div class="card dark">
+                <footer class="card-footer">
+                  <#Code
+                    language="elixir"
+                    module={{ SurfaceSiteWeb.Events.LiveButton }}
+                    line_range={{ 2..18 }}
+                    show_line_numbers
+                    selected_lines="5, 12"
+                  />
+                </footer>
+              </div>
+
+              <#Markdown>
+              We declared a required `click` event property that we use on the `<button>` tag with the `:on-click` directive.
+
+              Now let's see how to define and pass events to that stateless component.
+              We will use twice the `Button` component with two different handling functions that have been defined in a parent live component.
+              </#Markdown>
+
+              <div class="card dark">
+                <div class="card-content">
+                  <SurfaceSiteWeb.Events.LiveButton.Example
+                    id="surface_counter_2"
+                  />
+                </div>
+                <footer class="card-footer">
+                <#Code
+                  language="elixir"
+                  module={{ SurfaceSiteWeb.Events.LiveButton }}
+                  line_range={{ 20..42 }}
+                  show_line_numbers
+                  selected_lines="10-11, 16-18, 20-22"
+                />
+                </footer>
+              </div>
+
+              <#Markdown>
+                Remember that stateless components cannot handle events and do not have state.
+                Events can only be handled in a `LiveView` or `LiveComponent` so we will store the state in that kind of component.
+              </#Markdown>
+
+              <#Markdown>
+                ### Stateful component
+                In some case, you want to have a default behaviour that is handled by the component itself and you want to let the developer overrides the default implementation with a custom one. To implement a default behaviour, the component must implement an `handle_event/3` function, and so it requires to be stateful.
```

Comment:
```suggestion

                In some cases, you may want to have a default behaviour that is handled by the component itself and let the developer overrides the default implementation with a custom one. To implement a default behaviour, the component must implement an `handle_event/3` function, and so it requires to be stateful.
```

Reply:
Don't think I can update your suggestion, two small additional suggestions:

- overrides -> override
- it requires to be stateful -> it must be stateful